### DISCUSSION TITLE
Code for the MCU

### DISCUSSION THREAD

**Discussion by number-g** [PlummersSoftwareLLC/PETRock#27] [Q&A]
Hi,

Where is the code for the MCU?

Thanks

**Comment by rbergen** [comment]
I'm converting this to a Q&A discussion.

**Comment by rbergen** [comment]
@number-g I'm presuming you're talking about the device that does the spectrum analysis and outputs it to the PET/C64? That would be the [NightDriverStrip](https://github.com/PlummersSoftwareLLC/NightDriverStrip) repository. If I'm not mistaken, it concerned the Spectrum project (as defined in [platformio.ini](https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/platformio.ini) and [globals.h](https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/include/globals.h)), running on an M5StickC Plus, with `ENABLE_AUDIOSERIAL` set to 1.

**Reply by number-g** [reply]
Great, thanks!
